theme/theme.scss

```diff
@@ -37,4 +37,8 @@
 
 	// Center the image if its width is smaller than content's width.
 	margin: 0 auto;
+
+	// Make sure the image never exceeds the size of the parent container (#67).
+	max-width: 100%;
+	max-height: 100%;
```

Comment:
Given the editor's height is not restricted by CSS (parent container), then yes. I wouldn't be so sure there is no such case, though. OTOH, you're probably right.